app/src/main/java/com/example/drawingactivity/DialogUtils.kt

```diff
@@ -137,6 +135,223 @@ object DialogUtils {
             }
         }
     }
+
+    fun colorPickerBottomSheet(
+        context: Context,
+        previousColor: FloatArray,
+        /**
+         * Lambda that provides the string of the current color.
+         * Returns nothing. Used to save the hex value of the current color.
+         */
+        onSubmit: ((FloatArray?, String?) -> Unit)
```

Comment:
If I rotate, will this keep my current color values?

Reply:
adding a lambda to restore color on rotation